finance-admin/internal/server/post_upload.go

```diff
@@ -17,7 +18,21 @@ type UploadFormHandler struct {
 func (h *UploadFormHandler) render(v AppVars, w http.ResponseWriter, r *http.Request) error {
 	ctx := r.Context()
 
+	var err error
+	pisNumber := 0
```

Comment:
It has the same outcome but I think it would be more idiomatic to do `var pisNumber int` here instead, so save any confusion that the variable is just being created here and not given a default value.